crates/astria-composer/tests/blackbox/composer.rs

```diff
@@ -1,7 +1,14 @@
 use std::time::Duration;
 
-use ethers::types::Transaction;
+use ethers::{
+    types::Transaction,
+    utils::rlp::{
+        Decodable,
```

Comment:
Even though this is a test, we try to avoid polluting the namespace:
```suggestion
        Decodable as _,
```